cosmos/xion-testnet.json

```diff
@@ -0,0 +1,48 @@
+{
+  "rpc": "https://rpc.xion-testnet-1.burnt.com/",
+  "rest": "https://api.xion-testnet-1.burnt.com/",
+  "chainId": "xion-testnet-1",
+  "chainName": "Xion Testnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "XION",
+    "coinMinimalDenom": "uxion",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png"
+  },
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "xion",
+    "bech32PrefixAccPub": "xionpub",
+    "bech32PrefixValAddr": "xionvaloper",
+    "bech32PrefixValPub": "xionvaloperpub",
+    "bech32PrefixConsAddr": "xionvalcons",
+    "bech32PrefixConsPub": "xionvalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "XION",
+      "coinMinimalDenom": "uxion",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "XION",
+      "coinMinimalDenom": "uxion",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png",
+      "gasPriceStep": {
+        "low": 0.0,
```

Comment:
Let me do one more check. Is it okay for your chain and nodes to address zero gas?

Reply:
@ash-burnt

Reply:
Updated to 0.005, will change to zero if/when validators verify